### DISCUSSION TITLE
Uploading large (100+ MB) mp4 files onto Git Pages?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22302] [Other Feature Feedback, Questions, & Ideas]
<p>I am a complete novice to programming, but I am picking things up as I work on the project located at <a href="https://github.com/ialcanta/LingView" rel="nofollow noopener">https://github.com/ialcanta/LingView</a>. However I still haven’t figured this out:</p>
<p>I need to upload 100+ MB mp4s onto my Git Page but the Git Large File Storage info pages say that GLFS doesn’t work with Git Pages…</p>
<ol>
<li><strong>What are some alternatives to GLFS?</strong></li>
<li><strong>Or what are the methods of working with large files on Git Pages?</strong></li>
</ol>
<p>(I have heard of <em>git-annex, git-bigfiles, git-fat, git-media, git-bigstore,</em> and <em>git-sym,</em> but from what I understand those alternatives would either be outdated or too complex for me to understand/manage with my current programming knowledge. I would try embedding the videos and having them be hosted–if those are the right terms–on YouTube, but they need to be linked to the transcription software files (<em>.eaf</em>) that are the main focus of this project.)</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for reaching out.</p>
<p>In order to keep the GitHub service performing well for everyone, we don’t allow uploading large files (greater than 100MB) inside a Git repository. GitHub Pages is not intended to be used to distribute configuration files, large binaries, or other static assets. There are several low-cost services that are optimized for bulk storage and distributing large files. You can find more information on those services in <a href="https://help.github.com/articles/distributing-large-binaries" rel="nofollow noopener">our help documentation</a>.</p>
<p>I hope that helps!</p>

**Comment by ghost** [comment]
<p>Thank you for your help! I will check out the alternatives you suggested.</p>
